### PR TITLE
Add functional searchbar for both lost and found department of the app.

### PR DESCRIPTION
**Pull request by JannikUekoetter**
Currently, the "lost" and "found" sections are still on one page. For this reason there are also two searchbars. Please do not let this confuse you. This is still to be moved to different components.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 import styled from "styled-components";
 import LostItem from "../components/LostItem";
-import SearchbarLostItem from "../components/SearchbarLostItem";
 import FoundItem from "../components/LostItem";
 import { lostItems, foundItems } from "../components/Db";
+import { useState } from "react";
 
 export default function Home() {
+  const [search1, setSearch1] = useState("");
+  const [search2, setSearch2] = useState("");
```
Comment (NikRoe) [inline pages/index.js:10]: Your state and the corresponding setter function could have more descriptive names, like `[searchTerm, setSearchTerm]` for instance.

File: pages/index.js [hunk 2]
```diff
@@ -16,29 +19,48 @@ export default function Home() {
 
       <Main>
         <h1>My App</h1>
-        <SearchbarLostItem />
+        <div>
+          <h4>Search1</h4>
+          <input
+            placeholder="What are you looking for?"
+            onChange={(event) => setSearch1(event.target.value)}
+          ></input>
```
Comment (NikRoe) [inline pages/index.js:27]: Your `input` needs a corresponding `label` for accessibility reasons. More [info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)
For the label to then work properly, the input needs an `id` and the label an `htmlFor`.
The input is self-closing -> you don't need an opening and closing tag.

File: pages/index.js [hunk 3]
```diff
@@ -16,29 +19,48 @@ export default function Home() {
...
+              />
+            ))}
         </ul>
         <h2>Second Array of Dummy Data begins here:</h2>
+        <div>
+          <h4>Search2</h4>
+          <input
+            placeholder="What are you looking for?"
+            onChange={(event) => setSearch2(event.target.value)}
+          ></input>
```
Comment (NikRoe) [inline pages/index.js:48]: you `input` needs a corresponding `label` for accessibility reasons. More [info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

File: pages/index.js [hunk 4]
```diff
@@ -16,29 +19,48 @@ export default function Home() {
 
       <Main>
         <h1>My App</h1>
-        <SearchbarLostItem />
+        <div>
+          <h4>Search1</h4>
```
Comment (NikRoe) [inline pages/index.js:23]: Your page has an `h1`, an `h2` and two `h4`. You should not skip headings for your HTML to be more semantic and also for accessibility reasons. More [info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
If you want certain headings to appear visually in a different way, style them accordingly instead.

File: pages/index.js [hunk 5]
```diff
@@ -16,29 +19,48 @@ export default function Home() {
...
+                description={lostItem.description}
+                img={lostItem.img}
+                key={lostItem.id}
+                id={lostItem.id}
+              />
+            ))}
         </ul>
         <h2>Second Array of Dummy Data begins here:</h2>
+        <div>
+          <h4>Search2</h4>
```
Comment (NikRoe) [inline pages/index.js:44]: same as above (line 23)

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hey Jannik,

looks like a promising start!

I left some comments and remarks you might want to implement in your app regarding semantic HTML, accessibility and naming.

Keep it up 👍

**Review by kai-franke — APPROVED** [review]

**Pull request merged**